cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/model/InteractiveSecurityBuilder.java

```diff
@@ -130,14 +151,125 @@ public void buildRequest(CommandContext ctx, boolean buildRequest) throws Except
+            certAuthorityAccountName = caAccount;
+        }
+        //Let's Encrypt account KS and certAuthAccount prompts
+        if(useLetsEncrypt && caAccount == null) {
+            ctx.print("\nLet's Encrypt account key-store:");
+            //then account key store
+            while (accountKeyStoreFile == null) {
+                accountKeyStoreFile = commandInvocation.inputLine(new Prompt("File name (default " + defaultAccountKeyStoreFile + "): "));
+                if (accountKeyStoreFile != null && accountKeyStoreFile.length() == 0) {
+                    accountKeyStoreFile = defaultAccountKeyStoreFile;
+                }
+                //Check that we are not going to corrupt existing key-stores that are referencing the exact same file.
+                List<String> ksNames = ElytronUtil.findMatchingKeyStores(ctx, new File(accountKeyStoreFile), relativeTo);
+                if (!ksNames.isEmpty()) {
+                    throw new CommandException("Error, the file " + accountKeyStoreFile + " is already referenced from " + ksNames
+                            + " resources. Use " + SecurityCommand.formatOption(OPT_KEY_STORE_NAME) + " option or choose another file name.");
+                }
+            }
+
+            //then accountKeyStorePassword
+            while (accountKeyStorePassword == null) {
+                accountKeyStorePassword = commandInvocation.inputLine(new Prompt("Password (blank generated): "));
+                if (accountKeyStorePassword != null && accountKeyStorePassword.length() == 0) {
+                    accountKeyStorePassword = SSLSecurityBuilder.generateRandomPassword();
+                }
+            }
+
+            // then Let's Encrypt certificate authority account:
+            ctx.print("\nLet's Encrypt certificate authority account:");
+            //then account name
+            while (certAuthorityAccountName == null) {
+                certAuthorityAccountName = commandInvocation.inputLine(new Prompt("Account name (default " + defaultCertAuthorityAccountName + "): "));
+                if (certAuthorityAccountName != null && certAuthorityAccountName.length() == 0) {
+                    certAuthorityAccountName = defaultCertAuthorityAccountName;
+                }
+                //Check that we are not going to corrupt existing CertificateAccount.
+                List<String> caAccountNames = ElytronUtil.getCaAccountNames(ctx.getModelControllerClient());
+                if (caAccountNames.contains(certAuthorityAccountName)) {
+                    throw new CommandException("Error, the certificate authority account " + certAuthorityAccountName + " already exists.");
+                }
+            }
+
+            //then contactUrls / email
+            while (contactUrls == null || contactUrls.size() < 1) {
+                String emails = commandInvocation.inputLine(new Prompt("Contact email(s) [admin@example.com,info@example.com]: "));
```

Comment:
Since contact URL(s) are optional, we should allow it to not be specified. It's ok to show example contact URLs so a user knows how to format them but we shouldn't use these as default values.